### ISSUE TITLE
Keyboard navigation difficult on the site (no focus styling)

### ISSUE THREAD

**Issue by Seanom** [larrychristensen/orcpub#229]
Trying to navigate the site via keyboard is difficult in part due to the removal of outline styles.

As an advisory note if you are going to remove the outline property please provide an alternative :focus styling. Otherwise it's impossible to work out where you are on the page

```
.builder-option-dropdown:active, .builder-option-dropdown:focus {
    outline: 0;
}
```
and also this 
```
*:focus {
    /* outline: 0; */
}
```
